SingularityUI/app/controllers/Status.coffee

```diff
@@ -2,7 +2,7 @@ Controller = require './Controller'
 
 State = require '../models/State'
 
-StatusView = require '../views/status'
+StatusView = require('../views/status').default
```

Comment:
you should be able to delete this file entirely, since you now have `Status.es6` below